app/src/main/java/waiter/Router.java

```diff
@@ -0,0 +1,30 @@
+package waiter;
+
+import java.util.Map;
+
+public class Router {
+
+    Map<String, Map<String, String>> routes;
+
+    public Router() {
+        this.routes = Map.ofEntries(
+                Map.entry("GET /simple_get", Map.ofEntries(
+                        Map.entry("status", "200 OK"),
+                        Map.entry("headers", "Accept-Language: en-US\r\nContent-Length: 0"),
```

Comment:
I put it in to show format of the headers entry, but there is no necessity for it, I can take it out as a result!

Reply:
Gotcha. I would remove it since it isn't strictly necessary yet. You'll build out more of the header formatting soon enought ot meet future functional requirements.

Reply:
Sounds good 😄

Reply:
Addressed in [this commit](https://github.com/jonmanahan/waiter/pull/14/commits/3e780f258f08e75533484a8eec8eec26e229510b)